tests/suites/deploy/deploy_revision.sh

```diff
@@ -11,7 +11,7 @@ run_deploy_revision() {
 	wait_for "juju-qa-test" "$(charm_rev "juju-qa-test" 9)"
 
 	# check resource revision per channel specified.
-	juju resources juju-qa-test --format json | jq -S '.resources[0] | .[ "revision"] == "3"'
+	juju resources juju-qa-test --format json | jq -S '.resources[0] | .[ "revision"] == "1"'
```

Comment:
any idea why this changed?

Reply:
Not a clue

Reply:
I'm guessing they changed the `juju-qa-test` charm

Reply:
yup, it have no revision file now

Reply:
@barrettj12 @wallyworld @hpidcock unless someone on the team updated this charm, the revision shouldn't have changed.  
https://github.com/juju/juju/blob/2.9/testcharms/charm-hub/charms/juju-qa-test/README.md